sdk/samples/storage/blobs/src/blobs_client_example.c

```diff
@@ -21,12 +21,12 @@
 #include <azure/storage/az_storage_blobs.h>
 
 // Uncomment below lines when working with libcurl
-// #include <curl/curl.h>
```

Comment:
Undo all changes to this file.

Reply:
Alternatively, remove comments saying "uncomment below" - because you just uncommented, and for the reader it is unclear what is going on.

Reply:
yeah, missed to push the last commit, tks